scenario-api/src/main/scala/pl/touk/nussknacker/engine/build/ScenarioBuilder.scala

```diff
@@ -104,6 +104,9 @@ object ScenarioBuilder {
   def streamingLite(id: String) =
     new ProcessMetaDataBuilder(MetaData(id, LiteStreamMetaData()))
 
+  def withCustomMetaData(id: String, properties: Map[String, String]) =
```

Comment:
Why do you need that?

Reply:
Let's write a comment why we change production api or remove redundant properties verification

Reply:
I need that for the sake of UI validation. Based on our private discussion I will remove my factory method thanks to removal of metadata checks.